MAVProxy/mavproxy.py

```diff
@@ -908,10 +908,7 @@ def run_script(scriptfile):
         action='append',
         default=[],
         help='Load the specified module. Can be used multiple times, or with a comma separated list')
-    parser.add_option("--mav09", action='store_true', default=False, help="Use MAVLink protocol 0.9")
-    parser.add_option("--mav10", action='store_true', default=False, help="Use MAVLink protocol 1.0")
-    parser.add_option("--mav20", action='store_true', default=True, help="Use MAVLink protocol 2.0")
-    parser.add_option("--auto-protocol", action='store_true', default=False, help="Auto detect MAVLink protocol version")
+    parser.add_option("--mavversion", default="", help="Force MAVLink Version (1.0, 2.0). Otherwise autodetect version")
```

Comment:
Should really use `1.0`` as the default here.

Should use choices([...]) to limit the options.

Also, it could force 0.9, so the comment's not quite right.